src/Common/Event_Automator/Power_Automate/Power_Automate_Provider.php

```diff
@@ -686,6 +686,10 @@ public function add_refunded_edd_order_to_queue( $order_id, $refund_id, $all_ref
 	 * @param WC_Order $order      The instance of the order object.
 	 */
 	public function add_refunded_woo_order_to_queue( $order_id, $old_status, $new_status, $order ) {
+		if ( ! function_exists( 'tribe_tickets_get_ticket_provider' ) ) {
```

Comment:
While just adding a filter to a non-existent hook is a pretty small, why not just disable them all.

Also - instead of a specific function, let's disable based on the plugin (not) being loaded.